manifests/Lexikos/AutoHotKey/1.1.32.00.yaml

```diff
@@ -0,0 +1,15 @@
+Id: lexikos.autohotkey
+Version: 1.1.32.00
+Publisher: Lexikos
+Name: AutoHotKey
```

Comment:
Official AHK website names it `AutoHotkey`, without a capital 'K'

Reply:
Fixed in new commit as well